### DISCUSSION TITLE
Please Update widelands to release 1.1

### DISCUSSION THREAD

**Discussion by klaus-halfmann** [Homebrew/discussions#3924] [Writing Formulae/Casks]
I found https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/widelands.rb 
but I am not sure If I can just update it (there are some special tweaks included):

Release is https://github.com/widelands/widelands/releases/tag/v1.1 

OTOH we have a nasty bug we need to fix in some Release 1.1.1 or such: https://github.com/widelands/widelands/issues/5655

As I am a widelands developer You can contact me about Issues building widelands.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by klaus-halfmann** [comment]
There is a (stale) PR at https://github.com/Homebrew/homebrew-core/pull/113841 and thanks for your hint.
How can I get this going @SMillerDev

**Reply by SMillerDev** [reply]
Contribute to it, help the creator of the pull request out.

**Reply by klaus-halfmann** [reply]
I added a comment to the PR, waiting for the contributors to answer. 
Some part of the compile pipeline failed. But it is in some private repo at https://github.com/chenrui333/homebrew-core/tree/bump-widelands-1.1
If the fail to answer in a few day I will "use the force" and clone this into my private repo and start again from there.

**Reply by chenrui333** [reply]
@klaus-halfmann you can also create a new PR as well, we tried bunch of commits, but failed to build the artifact. Let me know whatever we can help.

**Comment by klaus-halfmann** [comment]
Some macOS Specific Bug was fixed https://github.com/widelands/widelands/issues/5655 wil eventuallly try again these day
